src/antispoof.c

```diff
@@ -192,6 +225,150 @@ static inline int antispoof_arp(struct pg_antispoof_state *state,
 	return -1;
 }
 
+void pg_antispoof_nd_enable(struct pg_brick *brick)
+{
+	pg_brick_get_state(brick, struct pg_antispoof_state)->nd_enabled =
+		true;
+}
+
+int pg_antispoof_nd_add(struct pg_brick *brick, uint8_t *ip,
+			struct pg_error **errp)
+{
+	struct pg_antispoof_state *state =
+		pg_brick_get_state(brick, struct pg_antispoof_state);
+	uint16_t n = state->nds_size;
+	struct nd *nd = &state->nds[n];
+
+	if (unlikely(n == ND_MAX)) {
+		*errp = pg_error_new("Maximal IPV6 reached");
+		return -1;
+	}
+
+	rte_memcpy(&nd->ip, ip, 16);
+	state->nds_size++;
+	return 0;
+
+}
+
+int pg_antispoof_nd_del(struct pg_brick *brick, uint8_t *ip,
+			struct pg_error **errp)
+{
+	struct pg_antispoof_state *state =
+		pg_brick_get_state(brick, struct pg_antispoof_state);
+	int index = -1;
+	uint16_t size = state->nds_size;
+	struct nd *nds = state->nds;
+
+	for (int16_t i = 0; i < size; i++) {
+		if (!memcmp(&nds[i].ip, ip, 16)) {
```

Comment:
I'm not sure there is a function for this in ip.h, but if not, instead of using memcmp can you add ip_cmd in ip.h please